### ISSUE TITLE
<Enrollment verification> from <My-Education-Benefits>

### ISSUE THREAD

**Issue by alexander-ferzola** [department-of-veterans-affairs/va.gov-team#33863]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more new unauthenticated (static) VA.gov pages
- [ ] Making revisions to one or more existing unauthenticated (static) VA.gov page
- [ x] Launching a new online form, app, or tool on VA.gov
- [ ] Making revisions to an existing online form, app, or tool on VA.gov
- [ ] Something else (please add details in the next question)

## What do you need help with?
  - [ x] Content support
  - [ x] IA support
  
*Please describe your need and provide any additional documents and/or links that will help us help you. If we need to add a react widget to a page, be sure to provide the code here.*

Following up on our midpoint review, the My Education Benefits (Digital GI Bill (DGIB)) Team is looking for Content support for a new feature within VA Education, In-App Enrollment Verification. In-App Enrollment Verification will allow Post-9/11 GI Bill students to verify their monthly enrollment information (attendance, scheduling, credit hours, etc.) which will allow them to get paid their monthly housing allowance, in accordance with a new piece of legislation, Section 1010. This feature will be available while authenticated through Va.Gov in order to continue receiving their monthly awards. Prior to implementation, the only ways for a Post-9/11 GI Bill student to complete their monthly enrollment verification is via text or email links. The authenticated experience for monthly enrollment verification will allow users to view a recap of all their prior verified months, future scheduled payments, and directly verify within the application. 

To this point, we have completed a design intent & midpoint review in the collaboration cycle and are seeking feedback from your team for:

- _Content Review_: End to end review of current content to confirm that our copy is user-friendly and accurate. Note that we are still working through the initial entry point designs (e.g., if they navigate to 'Education & Training', 'Manage Benefits', and then 'Verify School Enrollment') since we will need to consider flows here for different benefit types. 
-  _Recommendations for IA_: Would like IA Review and recommendations for where the app should live on Va.Gov. Current recommendation is to implement this new feature within Education & training >> Manage your benefits >> Verify Your School Enrollment. We would then anticipate that we would give them the option to pick which benefit they are verifying. Non-33 Chapters would redirect to WAVE (current state). Post 9-/11 GI Bill would redirect to our new feature. 

  
## Supporting Artifacts
*Please provide supporting artifacts as available.*

- Link to product outline: 
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/my-education-benefits/enrollment-verification/product-outline.md

- Link to or attach user flows:
- Link to prototype or mockups:

https://share.goabstract.com/6006cf1c-c40a-4bb4-912f-e4d05bb34dd9

- Link to any epics/issues that may be helpful:

## Will this work be going through the Collaboration Cycle?
- [ x] Yes
-  Completed the midpoint review, November 2021
- [ ] No

## When does this work need to be done?
- Estimated launch date: TBD
- Estimated staging review date: TBD
- Content and IA work needed by:  12/15

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [ x] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team
Leelah Holmes
Gaige Dickerson
Alex Ferzola
Audra Ayotte
Russell Lyons

- Team name: 
-	OCTO-DE product lead: Matt Self 
-	Product manager: Shamik Desai
-	OCTO-DE Design Lead: Ryan Thurlwell 

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#va-sitewide-content](https://dsva.slack.com/channels/va-sitewide-content) Slack channel and tag @RLHecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by mnorthuis** [comment]
@alexander-ferzola @LeelahHolmes Have you created any user flows for this work?

**Comment by alexander-ferzola** [comment]
> @alexander-ferzola @LeelahHolmes Have you created any user flows for this work?

Nothing specific past the collection links. Is there any question we can help with?

**Comment by mnorthuis** [comment]
Hi @alexander-ferzola @LeelahHolmes, I have a couple of questions for you

1. The mock-ups do not show a confirmation page.  When the user submits their verifications, a standard form flow would have a confirmation page to notify the user their work was successfully submitted.  I'm not sure what the intended destination and messaging is based on the screens provided.  
2. What does "Finish verifying your enrollments later" do? Does it just save their progress?  How does the user restart this flow if they had multiple verifications and time has passed and now have more?  Does the main enrollments page have messaging to show that they have verifications in progress?  
3. What happens for users that have multiple months to verify, but they attempt to start verification with the most recent month and haven't verified past months?  Is this flow even allowed? Does it take them through the multi-enrollments flow irregardless?  If so, how is that communicated to the user if they click on September 2021 but the first step of the flow has July 2021?  What is the purpose of multiple CTAs if they have to verify in order?


For awareness @bethpottsVADEPO and @RLHecht, this work will require a new static landing page that will need to support both this new tool for verifying enrollments for the Post 9/11 GI Bill benefits, but also will need to support an entry to WAVE for verifying enrollments of other edu benefits.   We currently link directly to WAVE, so that link will change to go to this new static page.

**Comment by alexander-ferzola** [comment]
Hi @mnorthuis, 

Thank you for the questions. 
1. When verifying a monthly enrollment, a user is brought to a summary screen (an application review) before submitting their enrollments. E.g. [](https://share.goabstract.com/6006cf1c-c40a-4bb4-912f-e4d05bb34dd9?collectionLayerId=db0ff19e-ba20-45f4-9178-2b419516d004&mode=design&sha=latest). After submitting, they would be brought back to the landing page that has all of their monthly enrollment verifications but the status for the newly verified months will have changed. As is, we wanted to limit the number of screens within the 'enrollment application' section, and previous feedback in the collaboration cycle did not think the screen was necessary. Are you recommending that there be an application confirmation screen after verifying enrollments that isn't just bringing them back to their monthly enrollments?

2. "Finish verifying your enrollments later" will bring them back to their main monthly enrollment verification landing page. Giving the user the option to go back to their verifications and resolve the enrollment discrepancy without submitting will mean there are fewer payments that are put on hold. I do not believe we anticipate it will save their progress (@LeelahHolmes/ @rlyons89  please confirm), because 1. the in-app monthly enrollment verification process isn't truly an application (previously had existed as a modal but was parsed into pages for accessibility) and 2. If a user is coming back in to verify, they will (hopefully) have a different yes/no for the monthly enrollment verification question. 

3. This was another recommendation that came out of the Design Collaboration Cycle. By rules, if a user clicks the verify monthly enrollment link, it must take them to their earliest enrollment to verify. The user can only verify first their oldest outstanding monthly enrollment, and it must be verified chronologically. CTAs that bring them to the same in-app enrollment verification was a recommendation to alleviate this confusion. 


Did my best to answer these questions but please let me know if it would be easier to hop on a call and explain any of this live! Appreciate all the support.

**Comment by rlyons89** [comment]
Confirming for #2 that we will not be saving anything in progress, a user will have to submit all of their enrollment verifications at once or start over the next time they come in.

**Comment by mnorthuis** [comment]
Thank you both for such speedy responses.  A few responses, happy to jump on a call if that would help as well. 

1.  I remember the convo about eliminating the review screen, but not the confirmation.  I'm not necessarily recommending you add in a screen, but you do need to provide some sort of confirmation messaging/notification so users know that the submission was successful.  
2.  For this, since it is not saving, the link language should be clear that they are essentially cancelling out of the flow and any responses they have provided. 
3.  Hmmm, I question this a bit.  If you click on a CTA for September and land on a page for August, that can be confusing.  Something needs to tell the user why.   At the very least this needs to be explained to the user through content.

**Comment by bethpottsVADEPO** [comment]
@alexander-ferzola Apologies for the delay in getting you content feedback! I'm attaching a Word doc with edits to the screens in the mockup. A few notes to summarize the types of edits I made:
1/ I changed passive to active voice throughout.
2/ I wanted to standardize one way of referring to the benefit rather than varying "awards" and "payments"--you'll see that I mostly went with "payments" since that's more specific and universally understood. Some Veterans might not know that "awards" means "payments."
3/ I removed italics since we don't use italics on our site per the style guide. It's a little harder to read on a screen.
4/ We try to limit use of bolding for emphasis, so I'm suggesting removing that in some places. 
5/ I did edit for our VA.gov voice, style, and plain language standards throughout. If you have any specific questions, please let me know!

[https://app.zenhub.com/files/133843125/3790a18d-e185-49d8-90b0-9b22af9a73cc/download](https://app.zenhub.com/files/133843125/3790a18d-e185-49d8-90b0-9b22af9a73cc/download)

**Comment by alexander-ferzola** [comment]
@bethpottsVADEPO @mnorthuis  thank you both for your support! Really appreciate it. We'll review all of this feedback and reach out about any questions.

**Comment by mnorthuis** [comment]
@alexander-ferzola Did you complete user research on this? Can you provide an update on where you are at and we can determine what work is needed for content and IA?

**Comment by alexander-ferzola** [comment]
Hi @mnorthuis! @bethpottsVADEPO did a content review for us back in December so I think we are set for now while we incorporate feedback into designs. We have upcoming usability testing planned and will circle back with those results. Thanks for the assistance!

**Comment by mnorthuis** [comment]
I would like to review screens prior to user testing to make sure we have any breadcrumbs and navigational elements established as accurately as possible.  Can you send me the current version and I will write up recommendations for the user test?  Thank y ou!

**Comment by alexander-ferzola** [comment]
> I would like to review screens prior to user testing to make sure we have any breadcrumbs and navigational elements established as accurately as possible. Can you send me the current version and I will write up recommendations for the user test? Thank y ou!

Sounds good, thank you for clarifying. We have designs in progress this week for the entry points and unauthenticated page (as well as the full online enrollment verification collection). We'll reach out next week with more collection links!

**Comment by mnorthuis** [comment]
From: "alexander.ferzola@accenturefederal.com" <alexander.ferzola@accenturefederal.com>
Date: Wednesday, January 26, 2022 at 12:09 PM
To: "mikki@adhocteam.us" <mikki@adhocteam.us>, "Potts, Beth" <Beth.Potts@va.gov>, "Hecht, Randi L. (Ad Hoc LLC)" <Randi.Hecht@va.gov>
Cc: Ricardo Da Silva <Ricardo.DaSilva@va.gov>, "Hurley, Tammy, VBAVACO" <tammy.hurley1@va.gov>, "Holbrook, Robert, VBAVACO" <Robert.Holbrook@va.gov>, "Self, Matthew" <Matthew.Self2@va.gov>, "audra.ayotte@accenturefederal.com" <audra.ayotte@accenturefederal.com>, "gaige.dickerson@accenturefederal.com" <gaige.dickerson@accenturefederal.com>, "leelah.holmes@accenturefederal.com" <leelah.holmes@accenturefederal.com>, "russell.lyons@accenturefederal.com" <russell.lyons@accenturefederal.com>
Subject: [EXTERNAL] DGIB Online Enrollment Verification - Usability Testing

 

Good afternoon, VA IA Team,

 

Mikki requested to review our DGIB Online Enrollment Verification for IA prior to usability testing. We’ve attached some links below to help:

[Entry points](https://app.abstract.com/share/7f89c674-7c36-411d-9acb-7d9951626c8c), where online enrollment verification will be accessed & live on Va.gov
Including: VA Homepage, Top Nav, Sidebar, Education and training homepage, and Unauthenticated screens
The [online enrollment verification collection](https://app.abstract.com/share/6006cf1c-c40a-4bb4-912f-e4d05bb34dd9), that follows workflow for a Chapter 33 student verifying their monthly enrollments via Va.gov
Planned Breadcrumbs:  Home > Education and training > Verify your school enrollments > Post-9/11 GI Bill enrollment verifications (after authenticating)
Link to our [Research Plan & Conversation Guide](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/my-education-benefits/enrollment-verification) for Usability Testing on GitHub
 

As a reminder, this is our first round of usability testing for enrollment verification, and we are looking to begin testing sessions in February. If there are any changes that you recommend before testing, please return feedback before EOD Friday, January 28th.

 

Please reach out if you have any questions!

 

 

Respectfully,

 

cidimage001.png@01D75946.A981FC50

Alexander Ferzola

Strategy & Consulting Analyst 

U.S. Department of Veterans Affairs Account

**Comment by mnorthuis** [comment]
IA feedback prior to user testing. 

**Research:**
- The research must include a task around findability of your new product on VA.gov, and that task should be asked before they see a prototype of the new product itself.  
  - They can either be started on the VA.gov home page, or whatever their browser home page is. 
  - Try to use as neutral of language as possible to not lead the participant to any specific area of the site or featured link.
  - Track what language they use when talking about the task as well as any search terms they may enter.

**User flows:**
Here is representation of the user flows based on your H1s and link labels used in your prototypes
![image](https://user-images.githubusercontent.com/20994159/151576059-1fd0e483-e855-4ebb-8a59-9b24b43a343b.png)

Recommendations:

- "Verify your enrollments" link can be clicked on from any month, but always requires the user to verify ALL of their enrollments. Work with accessibility on this link label to ensure the label is clear enough for screen readers to know what month they are selecting.  In addition, ensure this does not cause confusion and unnecessary cognitive load if a user clicks on a link for September, but the next page's heading is "August".
- "Finish later" link may infer that their progress is saved. This is the label we use for other applications when they want to save progress.  Work with sitewide content on this label to consider something closer to a "cancel" function.
- "Submit verifications" action doesn't result in a clear confirmation to the user.  Work with accessibility to ensure the link label and resulting heading is clear to all users that they have completed the process.  

**Breadcrumbs:**

- Breadcrumbs must accurately represent the hierarchy, and be present on all screens.   Some screens do not have an accurate breadcrumb (they differ between the entry points file and the visual design file), and some of the mobile screens are missing the breadcrumb component.  
- Breadcrumb segments must match the H1s.  As an example, based on the breadcrumbs I see in the mocks, the breadcrumb for the form flow would look like the following:  
  - Desktop: [Home]() > [Education and training]() > [Verify your school enrollments]() > [Enrollment verifications](url) > **Verify your enrollments**
  - Mobile:  < [Enrollment verifications](url)
- Breadcrumb standards are documented in the design system for reference.

**Entry points:**
- Home page and top nav:  There are limitations and rules around the number of links and placement of links for these areas.  Placement in either of these cannot be represented in user research unless it is reviewed and approved by the OCTO Sitewide PO and Sitewide content and IA.   Generally to be placed in these components, we need the following info:
  - Data supporting the new item is a top task for that benefit hub and would outrank other tasks already there
  - Data supporting which item would be removed (if already at max options)
  - If approved, Sitewide Content and IA will determine it's placement
- Education hub page: Given this is a task that they will need to do on a monthly basis, I would move this up the list to be the 3rd option under the "More resources" section.  
- Education left nav: The left nav should be in the same order as the hub page and include the same options.

**Comment by mnorthuis** [comment]
IA direction, including URLs, is documented here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/edu-enroll-verification.md

Once user testing is complete, please let me know, then we can modify as needed or finalize.

**Comment by mnorthuis** [comment]
@btruongafs @cesaresantiago Assigning you to this issue to ensure you have the content and IA information for the upcoming staging review. 

Also wanted to make sure that you knew that this work effort will require some redirects of legacy content once the new functionality and content pages are launched.  This is our standard process for any content brought over from the administration sites into the modernized experience so we do not have duplicate information to maintain or competing for search value.

We are aware of this existing page that will need to be redirected, but look to you to determine if there are any other legacy pages that should also be retired and redirected to the new content/feature.  

https://benefits.va.gov/gibill/isaksonroe/verification_of_enrollment.asp

Please be sure to open a [redirect ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=heyjoni%2C+jilladams%2C+RLHecht&labels=Sitewide+IA%2C+%E2%AD%90%EF%B8%8F+Public+Websites%2C+VA.gov+frontend&template=redirect-request.md&title=Redirect+Request) for these when you are ready.  The teams ask for 2 weeks notice to get these implemented, so this can be submitted at any time, with an implementation date to happen within a few days after the new content/feature launches. 

Let me know if you have any questions.

**Comment by mnorthuis** [comment]
@btruongafs Do you have everything you need from this ticket, are we good to close?
cc @RLHecht

**Comment by btruongafs** [comment]
@mnorthuis good to close!

**Comment by RLHecht** [comment]
Thanks all. Closing!
